### PR TITLE
[DEV-3665] Add Prayer Menu

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR adds a 4th nav bar item, and creates the initial menu structure on the main prayer tab. 

### How do I test this PR?
1. Run this branch locally.
2. You should see a new 4th nav bar item (praying man icon) in between "Discover" and "Connect".
3. Tap that icon.
4. You should be taken to the prayer tab. Here's what you should see:
Screenshot 2019-04-16 19 57 10
5. The menu is a horizontal scroll with 5 options:

- My Saved Prayers
- My Church
- My Campus
- My Community
- My Prayers

6. By tapping on each one of these items you should see a different description appear beneath it. 

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [DEV-3665](https://newspring.atlassian.net/browse/DEV-3665)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by richarddubay** [comment]
Full disclosure - I did not test this on Android. I ran into some issues when I opened this up in Android Studio and I wanted to make sure that I (a) disclosed that information and (b) asked for some help getting it up and running on the Android sim.

**Comment by richarddubay** [comment]
Also ... at some point we will have to get actual images and text from Comms for this.

File: packages/newspringchurchapp/src/ui/TileImage/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+TileImage.propTypes = {
+  image: GradientOverlayImage.propTypes.source,
+  isLoading: PropTypes.bool,
+  link: PropTypes.string,
+  onPressItem: PropTypes.func,
+  overlayColor: PropTypes.arrayOf(PropTypes.string),
+  text: PropTypes.string,
+};
+
+TileImage.defaultProps = {
```
Comment (IsaacHardy) [inline packages/newspringchurchapp/src/ui/TileImage/index.js:61]: Why is this needed?

**Review by IsaacHardy — CHANGES_REQUESTED** [review]
I think this looks good so far, but we need to move the My Prayers view to it's correct location now that we have the menu up. It's currently located under the connect tab.

**Review by redreceipt — CHANGES_REQUESTED** [review]
Add some TODOs to remove the stuff we won't need once the icon gets merged into core

File: packages/newspringchurchapp/src/ui/TileImage/index.js (hunk 1 above)
Comment (richarddubay) [reply packages/newspringchurchapp/src/ui/TileImage/index.js:61]: Not sure. This was copied over from TAP core. Maybe just to make sure that the text doesn't get accidentally set to something else?

**Comment by richarddubay** [comment]
@IsaacHardy Can we do the move of the "My Prayers" stuff as part of https://newspring.atlassian.net/browse/DEV-3693 so that this can get merged? I can add it as a subtask.

File: packages/newspringchurchapp/src/ui/TileImage/index.js (hunk 1 above)
Comment (IsaacHardy) [reply packages/newspringchurchapp/src/ui/TileImage/index.js:61]: /shrug

**Review by redreceipt — APPROVED** [review]

**Review by IsaacHardy — APPROVED** [review]
